plugin/src/_rules/position.js

```diff
@@ -0,0 +1,23 @@
+import { handler as h, insetMap } from '#utils';
+
+function handleInsetValue(v, { theme }) {
+    return theme.spacing?.[v] ?? h.fraction.auto(v); // TODO: warn if no value
+}
+function handleInsetValues([, d, v], ctx) {
+    const r = handleInsetValue(v, ctx);
+    if (r != null && d in insetMap)
+        return insetMap[d].map(i => [i.slice(1), r]);
+}
+export const insets = [
+    [/^inset-(.+)$/, ([, v], ctx) => ({ inset: handleInsetValue(v, ctx) }),
+        {
+            autocomplete: [
+                '(inset-$spacing',
+                '(inset-<direction>-$spacing',
```

Comment:
@pearofducks Should we also allow autocomplete with "full", "auto", "1/2", "1/4"?

Reply:
It seems like we might already be including that here? At least the original preset-mini autocomplete code doesn't have any special consideration for those?

I think it's a great thing to note down somewhere (a 'test autocomplete' task?) or keep in our mind for when we do check that though!